int32 actuator_id
---
float64 voltage
float64 current_scale
float64 velocity_scale
float64 actuator_temperature
float64 inverter_temperature